lib/shoryuken/manager.rb

```diff
@@ -12,7 +13,7 @@ class Manager
 
     def initialize(condvar)
       @count  = Shoryuken.options[:concurrency] || 25
-      @queues = Shoryuken.queues.dup.uniq
+      @polling_strategy = Shoryuken.options[:polling_strategy].new(Shoryuken.queues)
```

Comment:
We can't break the config in minor versions. WeightedRoundRobin has to be set as default!